core/bessctl.cc

```diff
@@ -1147,6 +1156,10 @@ class BESSControlImpl final : public BESSControl::Service {
       response->set_name(module->name());
     }
 
+    if (module->OnEvent(bess::Event::PreResume) != ENOTSUP) {
```

Comment:
This event is somewhat spurious, since `PreResume` event is not really happening here. I think it should be fine for most module handlers, but how about,
1. By default, we add all new modules to `event_modules` (without `if` here).
1. `ENOTSUP` is checked in the loop over `event_modules`.

Reply:
Good point. Will change